src/Resources/doc/index.rst

```diff
@@ -958,6 +958,123 @@ For better security of your site please use ``no-referrer``, ``same-origin``,
+* ``['self']`` - Allows the feature for the same origin as the document
+* ``['*']`` - Allows the feature for all origins
+* ``['src']`` - Allows the feature for the same origin as the src attribute (iframe context)
+* ``['https://example.com']`` - Allows the feature for specific origins (URLs must be quoted)
+
+Common permissions policies:
+
+.. code-block:: yaml
+
+    # config/packages/nelmio_security.yaml
+    nelmio_security:
+        permissions_policy:
+            # Media permissions
+            camera: []
+            microphone: []
+
+            # Location and sensors
+            geolocation: []
+            accelerometer: []
+            gyroscope: []
+            magnetometer: []
+
+            # Privacy features
+            interest_cohort: []           # Disable FLoC tracking
+
+            # Payment and authentication
+            payment: ['self']
+            publickey_credentials_get: ['self']
+
+            # Display and interaction
+            fullscreen: ['self']
+            picture_in_picture: ['self']
+            autoplay: []
+
+Available directive names (automatically converted from snake_case to kebab-case):
+
+* ``accelerometer``
+* ``ambient_light_sensor`` (becomes ``ambient-light-sensor``)
+* ``autoplay``
+* ``battery``
+* ``camera``
+* ``cross_origin_isolated`` (becomes ``cross-origin-isolated``)
+* ``display_capture`` (becomes ``display-capture``)
+* ``document_domain`` (becomes ``document-domain``)
+* ``encrypted_media`` (becomes ``encrypted-media``)
+* ``execution_while_not_rendered`` (becomes ``execution-while-not-rendered``)
+* ``execution_while_out_of_viewport`` (becomes ``execution-while-out-of-viewport``)
+* ``fullscreen``
+* ``gamepad``
+* ``geolocation``
+* ``gyroscope``
+* ``hid``
+* ``idle_detection`` (becomes ``idle-detection``)
+* ``interest_cohort`` (becomes ``interest-cohort``)
+* ``keyboard_map`` (becomes ``keyboard-map``)
+* ``magnetometer``
+* ``microphone``
+* ``midi``
+* ``navigation_override`` (becomes ``navigation-override``)
+* ``payment``
+* ``picture_in_picture`` (becomes ``picture-in-picture``)
+* ``publickey_credentials_get`` (becomes ``publickey-credentials-get``)
+* ``screen_wake_lock`` (becomes ``screen-wake-lock``)
+* ``serial``
+* ``sync_xhr`` (becomes ``sync-xhr``)
+* ``usb``
+* ``web_share`` (becomes ``web-share``)
+* ``xr_spatial_tracking`` (becomes ``xr-spatial-tracking``)
+
+.. caution::
+
+    Some directive names are experimental and may not be supported by all browsers.
+    Using unsupported directives will generate console warnings in browsers like Chrome.
+
+Browser compatibility:
+
+* **Chrome 88+**: Full support
+* **Edge 88+**: Full support
+* **Firefox**: Not supported (header is skipped automatically)
+* **Safari**: Not supported (header is skipped automatically)
+
+The bundle automatically detects Firefox and Safari user agents and skips sending
+the ``Permissions-Policy`` header to avoid compatibility issues.
```

Comment:
Is this really wise? I'd say we rather send it and hope the browsers implement it at some point. Otherwise it'll require a bundle update whenever support is added, which can easily be missed and take some months/years to end up live.

Reply:
No idea. but i guess you are right :)